var/spack/repos/builtin/packages/hipblas/package.py

```diff
@@ -160,10 +160,10 @@ def cmake_args(self):
 
         if self.spec.satisfies("^cmake@3.21.0:3.21.2"):
             args.append(self.define("__skip_rocmclang", "ON"))
-        if self.spec.satisfies("@:5.1.3"):
-            args.append("-DCMAKE_MODULE_PATH={0}".format(self.spec["hip"].prefix.cmake))
+        if self.spec.satisfies("@:5.1"):
+            args.append(self.define("CMAKE_MODULE_PATH", self.spec["hip"].prefix.cmake))
         elif self.spec.satisfies("@5.2.0:"):
-            args.append(self.define("BUILD_FILE_REORG_BACKWARD_COMPATIBILITY", "ON"))
+            args.append(self.define("BUILD_FILE_REORG_BACKWARD_COMPATIBILITY", "True"))
```

Comment:
Not the string `"True"` but the value `True`

Reply:
Changed to value True instead of string "True"